<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ongletCalcul</class>
 <widget class="QMainWindow" name="ongletCalcul">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>994</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coordonnées approchées</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/approchCoords.png</normaloff>img/approchCoords.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>60</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget {
    background-color:rgb(255, 255, 255);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Calcul</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="spinBoxViLimit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>268</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="runCalcul">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 218, 160);
border-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="parcourirPoints">
         <property name="text">
          <string>Parcourir</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_69">
         <property name="text">
          <string>Dossier des pts calculés</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_66">
         <property name="text">
          <string>Observations</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="pathObs">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="parcourirRes">
         <property name="text">
          <string>Parcourir</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="resDirPath">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Résidus limite [m]</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="pathPoints">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="parcourirObs">
         <property name="text">
          <string>Parcourir</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_67">
         <property name="text">
          <string>Points connus</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Recherche</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="planSysLoc">
      <property name="styleSheet">
       <string notr="true">border-color: rgb(98, 140, 255);</string>
      </property>
      <property name="title">
       <string>Plan du déroulement du calcul</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,1,0,0">
       <item row="1" column="0" colspan="2">
        <widget class="PlotWidget" name="plotLive" native="true">
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0" columnstretch="0,1,0">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBoxNomsPoints">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Afficher les noms de points</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>493</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1" colspan="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>950</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="coordinatesOfClick">
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" rowspan="2">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Détails</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5" rowstretch="0" columnstretch="0">
          <item row="0" column="0">
           <widget class="QTextEdit" name="textInfoLog">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>120</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Légende</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 255);</string>
            </property>
            <property name="text">
             <string>Points connus</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 181, 0);</string>
            </property>
            <property name="text">
             <string>Points de base du système à calculer </string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 0, 0);</string>
            </property>
            <property name="text">
             <string>Points nouveaux du système</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>58</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QSlider" name="horizontalSlider">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="sliderPosition">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
